docs/development.rst

```diff
@@ -33,21 +33,12 @@ If you choose to install `starknet-devnet-rs <https://github.com/0xSpaceShard/st
 
 In order to be able to run tests on testnet and integration networks (``starknet_py/tests/e2e/tests_on_networks/``), you must set some environmental variables:
 
-    - ``GOERLI_INTEGRATION_RPC_URL``
-    - ``GOERLI_TESTNET_RPC_URL``
     - ``SEPOLIA_INTEGRATION_RPC_URL``
-    - ``SEPOLIA_TESTNET_RPC_URL``
```

Comment:
SEPOLIA_RPC_URL is missing

Reply:
no, it's here https://github.com/software-mansion/starknet.py/pull/1328/files#diff-9b171ca00cb885ccd9c997cb6f65fbb26d7eae92bf54bd7581b83c4541c156f2R37

Reply:
ahh shiet, I misinterpreted the diff, my bad sorry ~